lib/runtime/wasmer/exports_test.go

```diff
@@ -394,16 +393,17 @@ func buildBlock(t *testing.T, instance runtime.Instance) *types.Block {
 	require.NoError(t, err)
 
 	// decode inherent extrinsics
-	exts, err := scale.Decode(inherentExts, [][]byte{})
+	var exts [][]byte
+	err = scale.Unmarshal(inherentExts, &exts)
 	require.NoError(t, err)
 
 	// apply each inherent extrinsic
-	for _, ext := range exts.([][]byte) {
-		in, err := scale.Encode(ext) //nolint
-		require.NoError(t, err)
+	for _, ext := range exts {
+		in, er := scale.Marshal(ext)
+		require.NoError(t, er)
 
-		ret, err := instance.ApplyExtrinsic(append([]byte{1}, in...))
-		require.NoError(t, err, in)
+		ret, e := instance.ApplyExtrinsic(append([]byte{1}, in...))
+		require.NoError(t, e, in)
```

Comment:
nit but i think errors in go should always be named `err`, so you can add a `//nolint` if that causes lint issues